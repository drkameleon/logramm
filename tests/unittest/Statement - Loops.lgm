i = 0;
@ i<10: {
	out i;
	i += 1;
}

@ x->[1..10] : out x;

@ i,x->[1..10] : out "{:i} --> {:x}";

arr = ["one","two","three"];

@ item->arr: out item;

dict = [
	[ .name : "user1", .age : 1 ],
	[ .name : "user2", .age : 2 ]
];

@ row->dict : out row;

@ key,value->dict : out "{:key} --> {:value}";

dict = [ .name : "user1", .age : 1 ];

@ row->dict : out row;

@ key,value->dict : out "{:key} --> {:value}";