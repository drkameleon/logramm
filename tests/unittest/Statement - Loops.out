0
1
2
3
4
5
6
7
8
9
1
2
3
4
5
6
7
8
9
0 --> 1
1 --> 2
2 --> 3
3 --> 4
4 --> 5
5 --> 6
6 --> 7
7 --> 8
8 --> 9
one
two
three
["name":"user1", "age":1]
["name":"user2", "age":2]
0 --> ["name":"user1", "age":1]
1 --> ["name":"user2", "age":2]
name
age
name --> user1
age --> 1
